### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by schirevko**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -47,8 +47,18 @@ EOF
 $ terraform init
 ```
 
+When submitting PR make sure that it has updates to CHANGELOG.md. Add description of changes under last version with "(unreleased)" mark.
+
+Documenting provider
+--------------------
+Provider documentation is generated automatically when updates are merged into master branch.
```
Comment (ftersin) [inline README.md:54]: PR with renewed provider documentation is generated...

File: README.md [hunk 2]
```diff
@@ -47,8 +47,18 @@ EOF
...
+To update documentation contents, please, update "description" field of necessary resource/data_source schema.
```
Comment (ftersin) [inline README.md:55]: and create/modify templates.

Also place this as the first sentence of the chapter.

File: README.md [hunk 3]
```diff
@@ -47,8 +47,18 @@ EOF
 $ terraform init
 ```
 
+When submitting PR make sure that it has updates to CHANGELOG.md. Add description of changes under last version with "(unreleased)" mark.
```
[the remark this answers is not in the document]
Comment (schirevko) [reply README.md]: Added

File: README.md (hunk 1 above)
Comment (schirevko) [reply README.md]: Rewrote

File: README.md (hunk 2 above)
Comment (schirevko) [reply README.md]: Fixed

**Review by ftersin — APPROVED** [review]

**Pull request merged**